satellite/console/consoleweb/server.go

```diff
@@ -247,107 +248,107 @@ func (s *Server) createRegistrationTokenHandler(w http.ResponseWriter, req *http
 }
 
-func (s *Server) passwordRecoveryHandler(w http.ResponseWriter, req *http.Request) {
-	ctx := req.Context()
+func (server *Server) passwordRecoveryHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
-	recoveryToken := req.URL.Query().Get("token")
+	recoveryToken := r.URL.Query().Get("token")
 	if len(recoveryToken) == 0 {
-		s.serveError(w, req)
+		server.serveError(w, r)
 		return
 	}
 
-	switch req.Method {
+	switch r.Method {
 	case http.MethodPost:
-		err := req.ParseForm()
+		err := r.ParseForm()
 		if err != nil {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
-		password := req.FormValue("password")
-		passwordRepeat := req.FormValue("passwordRepeat")
+		password := r.FormValue("password")
+		passwordRepeat := r.FormValue("passwordRepeat")
 		if strings.Compare(password, passwordRepeat) != 0 {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
-		err = s.service.ResetPassword(ctx, recoveryToken, password)
+		err = server.service.ResetPassword(ctx, recoveryToken, password)
 		if err != nil {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
-		http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "static", "resetPassword", "success.html"))
+		http.ServeFile(w, r, filepath.Join(server.config.StaticDir, "static", "resetPassword", "success.html"))
 	case http.MethodGet:
-		t, err := template.ParseFiles(filepath.Join(s.config.StaticDir, "static", "resetPassword", "resetPassword.html"))
+		t, err := template.ParseFiles(filepath.Join(server.config.StaticDir, "static", "resetPassword", "resetPassword.html"))
 		if err != nil {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 
 		err = t.Execute(w, nil)
 		if err != nil {
-			s.serveError(w, req)
+			server.serveError(w, r)
 			return
 		}
 	default:
-		s.serveError(w, req)
+		server.serveError(w, r)
 		return
 	}
 }
 
-func (s *Server) cancelPasswordRecoveryHandler(w http.ResponseWriter, req *http.Request) {
-	ctx := req.Context()
+func (server *Server) cancelPasswordRecoveryHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
 	defer mon.Task()(&ctx)(nil)
-	recoveryToken := req.URL.Query().Get("token")
+	recoveryToken := r.URL.Query().Get("token")
 	if len(recoveryToken) == 0 {
-		http.Redirect(w, req, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
+		http.Redirect(w, r, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
 	}
 
 	// No need to check error as we anyway redirect user to support page
-	_ = s.service.RevokeResetPasswordToken(ctx, recoveryToken)
+	_ = server.service.RevokeResetPasswordToken(ctx, recoveryToken)
 
-	http.Redirect(w, req, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
+	http.Redirect(w, r, "https://storjlabs.atlassian.net/servicedesk/customer/portals", http.StatusSeeOther)
```

Comment:
Would it make sense to make this a more global variable for the URL?

Reply:
Something like `server.service.serviceURL`

Reply:
This probably should be a config option with the default set to that.